countNumberOfLowercase.js

```diff
@@ -0,0 +1,13 @@
+export const typeOfArgumentErrorMessage = "There should be a string!"
+
+export function countNumberOfLowercase(string) {
+    let count = 0;
+    if (typeof string != "string") { throw new Error(typeOfArgumentErrorMessage) } else {
```

Comment:
You don't have to use if/else here. Only if is enough because you have the return. 
Learn more here: https://forum.freecodecamp.org/t/the-return-early-pattern-explained-with-javascript-examples/19364

Reply:
done